frankhn/questioner-challenge
# creating application server

#### What does this PR do?
This PR allow the App to get started on a created node server
#### Description of Task to be completed?
This is the first point to reach the app
#### How should this be manually tested?
Try to start the server in the command line run 
npm start to check if the server is learning
#### What are the relevant pivotal tracker stories?
Creating server for the application  #163332740
